### PR TITLE
[draft] coordinator rpc support

### PR DESCRIPTION
**Pull request by icezohu**

### CODE REVIEW

File: apps/indexer-coordinator/src/project/project.model.ts [hunk 1]
```diff
@@ -160,6 +267,10 @@ export class ProjectEntity {
   @Field()
   queryEndpoint: string; // endpoint of query service
 
+  @Column('jsonb', { default: {} })
+  @Field()
+  serviceEndpoints: IProjectNetworkEndpoints | IProjectRpcEndpoints;
```
Comment (ianhe8x) [inline apps/indexer-coordinator/src/project/project.model.ts:272]: The keys of endpoints are decided by project type + chainFamily when project is rpc.
```suggestion
  serviceEndpoints: Record<string, string>;
```

File: apps/indexer-coordinator/src/project/project.model.ts [hunk 2]
```diff
@@ -118,6 +118,81 @@ export class ProjectAdvancedConfig implements IProjectAdvancedConfig {
...
+  cpu: number;
+  @Field(() => Int)
+  memory: number;
+}
+
+@InputType('ProjectRpcConfigInput')
+@ObjectType('ProjectRpcConfig')
+export class ProjectRpcConfig implements IProjectRpcConfig {
+  @Field()
+  rpcFamily: string;
```
Comment (ianhe8x) [inline apps/indexer-coordinator/src/project/project.model.ts:193]:
```suggestion
  rpcFamily: string[];
```

File: apps/indexer-coordinator/src/project/project.model.ts [hunk 3]
```diff
@@ -172,16 +283,29 @@ export class ProjectEntity {
   @Field(() => ProjectAdvancedConfig)
   advancedConfig: ProjectAdvancedConfig;
 
+  @Column('jsonb', { default: {} })
+  @Field(() => ProjectConfig)
```
Comment (ianhe8x) [inline apps/indexer-coordinator/src/project/project.model.ts:287]: Don't think you can use db entity for graphql model definition.

**Comment by mzxyz** [comment]
Should this branch generate and merge into `develop` branch?

**Comment by icezohu** [comment]
> Should this branch generate and merge into `develop` branch?

ok, will do.

**Pull request closed without merging**